HelpSource/Tutorials/Getting-Started/02-First-Steps.schelp

```diff
@@ -96,45 +96,41 @@ line 3 char 10:
 
 Usually the problem actually occurs a little before that, so that's where you should look.  In this case, it's the lack of a semi-colon at the end of the previous line.  Note that each line of code ends normally with a semi-colon.  This is how you separate lines of code in SC. Since we didn't have a semi-colon between the two lines we have gotten an error. 
 
-Using semi-colons it's possible to have more than one line of code in the same line of text. This can be handy for execution.
-
 code::
 "Call me ".post; "Ishmael?".postln;
 ::
 
 Note also, having an extra semi-colon at the very end of the last piece of code does not hurt and is tolerated by SC for reasons of convenience. 
 
-A couple of more notes about the post window. It's very useful to be able to see it, but sometimes it can get hidden behind other windows. You can bring it to the front at any time by holding down the Command key, and pressing \ . The Command key is the one with the ⌘ symbol on it.
-
-By convention this kind of key sequence is written Cmd - \
+A couple of more notes about the post window. It's very useful to be able to see it, but sometimes it can get hidden behind other windows. You can bring it to the front at any time by pressing Cmd-\.
 
-As well, sometimes the post window becomes full of stuff and hard to read. You can clear it at any time by pressing Cmd-shift-p (hold down the command key and the shift key, and then press p).
+Other times, the post window becomes full of text and hard to read. You can clear it at any time by pressing Cmd-Shift-P.
```

Comment:
Can this be changed to `Ctrl-Shift-P` for consistency?

Reply:
I think this one still needs changing? Looks like you have corrected it somewhere else that I had missed (thanks for that!)

Reply:
haha, sweet! Good distributed coverage on our part 😆